src/lib.rs

```diff
@@ -198,6 +199,12 @@ pub use log::{
     unset_print_handler, unset_printerr_handler, LogHandlerId, LogLevel, LogLevels,
 };
 
+#[cfg(any(feature = "log", feature = "dox"))]
+#[macro_use]
+mod bridged_logging;
+#[cfg(any(feature = "log", feature = "dox"))]
```

Comment:
Does this work without declaring a feature for this in `Cargo.toml`? The optional dependency implicitly declares a feature?

Reply:
From what I could get: dependencies and features share the same namespace in Cargo.toml, so any optional dependency is implicitly also something you can specify as a feature.

If you are worried about discoverability, I think we can rename the `log` dependency giving it an alias, and then have a separate feature with the `log` name.

Reply:
As long as `cargo build --features log` works, this is ok for me. Discoverability of features is a whole other story and also suboptimal for other features already. For now this just has to be clear from the documentation.

Reply:
I added some notes in the documentation, now it should be clearer.